src/Aspire.Hosting.Azure.Sql/PublicAPI.Unshipped.txt

```diff
@@ -1,2 +1,12 @@
 #nullable enable
-
+Aspire.Hosting.Azure.AzureSqlDatabaseResource
+Aspire.Hosting.Azure.AzureSqlDatabaseResource.AzureSqlDatabaseResource(string! name, string! databaseName, Aspire.Hosting.Azure.AzureSqlServerResource! parent) -> void
+Aspire.Hosting.Azure.AzureSqlDatabaseResource.ConnectionStringExpression.get -> Aspire.Hosting.ApplicationModel.ReferenceExpression!
+Aspire.Hosting.Azure.AzureSqlDatabaseResource.DatabaseName.get -> string!
+Aspire.Hosting.Azure.AzureSqlDatabaseResource.Parent.get -> Aspire.Hosting.Azure.AzureSqlServerResource!
+Aspire.Hosting.Azure.AzureSqlServerResource.AzureSqlServerResource(string! name, System.Action<Aspire.Hosting.ResourceModuleConstruct!>! configureConstruct) -> void
```

Comment:
@mitchdenny @davidfowl - the nominclature here is really hard. Is this what we want?

* `AzureSqlServerResource` (existing) - The "Server" resource
* `AzureSqlDatabaseResource` (new) - The child "database" resource.

Reply:
The official service names don't help much :) I think AzureSqlDatabaseResource and AzureSqlServerResource is probably the best option.